VDA5050_EN.md

```diff
@@ -751,6 +753,34 @@ y |  | float64 | Y coordinate described in the world coordinate system.
 *weight* |  | float64 | Range: (0 ... infinity)<br><br>The weight, with which this control point pulls on the curve.<br>When not defined, the default will be 1.0.
 } |  |  |
 
+Object structure | Unit | Data type | Description 
+---|---|---|---
+**corridorPoint** { |  | JSON-object |  
+x |  | float64 | X coordinate described in the world coordinate system. 
+y |  | float64 | Y coordinate described in the world coordinate system.
+} |  |  |
+
+
+### <a name = "Corridor"></a> 6.7.1 Corridor
+
+For a vehicle, which plans autonomically the path from one node to the next node, the optional corridor object defines the boundaries in which the vehicle is allowed to operate. In contrast to an allowed deviation the corridor defines the boundaries which are not only valid for the tool center pointer of the vehicle, but they are also valid for every part of the vehicle including the load. If there is no need to avoid an obstacle the vehicle shall drive on or near by the current edge. 
+
+![Figure 16 Corridor with boundaries](./assets/Corridor-1a.png)
+>Figure 16 Corridor with boundaries.
+
+The corridor object defines a simple polygone (no self intersection, no holes) and the points shall be listed counter-clock wise. The coordinates of the points are inside the coordinate system of the edge start node. The boundaries of a single corridor polygon shall be defined in a way that the polygons of two consecutive edges overlaps so that the vehicle can travel form one edge to the next and the nodes of the order can be reached by the vehicle without disregarding the corridor boundaries (see figure 17). Polygons of non released edges aren't part of the current corridor.
+A vehicle which is pushed back manually on a traversed or not released edge is outside the corridor, therefore outside the allowed navigation space and isn't allowed to move. The union of all corridor polygons of the current base defines the navigation space (see figure 18).
+
+![Figure 17 Three edges with their current corridor defined by a simple polygon.](./assets/Polygon1.png)
+>Figure 17 Three edges with their current corridor defined by a simple polygon.
+
+The motion control software of the vehicle shall check permanently if a part of the vehicle or of its load is outside of the corridor. If this is the case the vehicle shall stop, because it is outside of the allowed navigation space, and to report an appropriate error. The MC can decide if a user interaction is necessary or if the vehicle can continue driving by canceling the current and sending a new order to the AMR with corridor information which allows the vehicle to move again.
```

Comment:
"Specific error": I have the feeling the panel is hard put to specify specific errors (in general), but I can bring this into discussion.
"Instant action": Everyone is free to define their specific instant actions. However this action would change the base of an order which is otherwise strictly forbidden. Putting such an action into the standard would made the standard inconsistent. I think it's not a good idea. If more flexibility is necessary the zone concept (still in an early discussion) is more suitable.

Reply:
An action like that  would not invalidate anything. It would just decrease the constraints on the agv/amr.

I do not see the point in adding the requirement for a feedback to the fleet manager without at least specifying  how this feedback is supposed to look like. Otherwise the agv/amr vendor is forced to adapt to any fleet manager or the fleet manager is supposed to adapt to any vendor. The standard suffers already from this ambiguity in many places (imo).